pkg/errors/errors.go

```diff
@@ -167,18 +209,26 @@ func (l *LintRuleErrorsList) ConvertToError() error {
 		}
 
 		builder.WriteString(fmt.Sprintf(
-			"%s%s\n\tMessage\t- %s\n\tObject\t- %s\n\tModule\t- %s\n",
+			"%s%s\n\tMessage\t- %s\n\tModule\t- %s\n",
 			emoji.Sprintf(":monkey:"),
 			color.New(color.FgHiBlue).SprintfFunc()("[#%s]", err.ID),
 			color.New(msgColor).SprintfFunc()(err.Text),
-			err.ObjectID,
 			err.Module,
 		))
 
+		if err.ObjectID != "" && err.ObjectID != err.Module {
+			builder.WriteString(fmt.Sprintf("\tObject\t- %s\n", err.ObjectID))
+		}
 		if err.ObjectValue != nil {
 			value := fmt.Sprintf("%v", err.ObjectValue)
 			builder.WriteString(fmt.Sprintf("\tValue\t- %s\n", value))
 		}
+		if err.FilePath != "" {
+			builder.WriteString(fmt.Sprintf("\tFilePath\t- %s\n", err.FilePath))
```

Comment:
There may be variations here, as we output exactly what we were given. And if you do not store the module directory separately in errors, it is not possible to normalize the paths to relative